src/cgame/CombatFeedback.cpp

```diff
@@ -294,24 +295,26 @@ static void DrawDamageIndicator(DamageIndicator *di)
 		text = "!" + text;
 	total_width = width * text.length();
 
-	x = di->pos2d.x() - total_width / 2;
-	y = di->pos2d.y() - height / 2;
+	x = di->pos2d[ 0 ] - total_width / 2;
+	y = di->pos2d[ 1 ] - height / 2;
 
 	for (size_t i = 0; i < text.length(); i++) {
 		int glyph;
-		Vec2 st0, st1;
+		vec2_t st0, st1;
 
 		if (text[i] >= '0' && text[i] <= '9')
 			glyph = text[i] - '0';
 		else
 			glyph = 10;
 
-		st0 = Vec2((glyph % 4) * 0.25f, (glyph / 4) * 0.25f);
-		st1 = st0 + Vec2(0.25f, 0.25f);
+		st0[ 0 ] = ( glyph % 4 ) / 4.f;
+		st0[ 1 ] = ( glyph % 4 ) / 4.f;
```

Comment:
the original code did that:
```suggestion
		st0[ 1 ] = ( glyph / 4 ) / 4.f;
```